pkgs/applications/misc/bottles/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, fetchFromGitHub
+, meson, ninja, pkg-config, wrapGAppsHook
+, desktop-file-utils, gsettings-desktop-schemas, libnotify
+, python3Packages, gettext
+, appstream-glib, gdk-pixbuf, glib, gobject-introspection, gspell, gtk3
+, steam-run-native
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "bottles";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "bottlesdevs";
+    repo = pname;
+    rev = "b0b5a259d3bb5cff89cd6dd7e7be3e6fcf3c7776";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
The releases are tagged without a prefixed 'v'. :smile:

Reply:
Didn't check that closely.